src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -316,6 +317,10 @@ public SnapshotBuilderGetConceptHierarchyResponse getConceptHierarchy(
     return new SnapshotBuilderGetConceptHierarchyResponse().result(List.copyOf(parents.values()));
   }
 
+  public SnapshotAccessRequestResponse rejectSnapshotAccessRequest(UUID id) {
```

Comment:
This class also has `createSnapshotAccessRequest()` and `enumerateSnapshotAccessRequests()`. Given that these are the only kinds of requests that this class handles, and `snapshot` is already in the class name, would shorter names work here instead? For example: `enumerateRequests`, `createRequest`, `rejectRequest`.

Reply:
I think I am for that!